language/stdlib/modules/VASP.move

```diff
@@ -64,8 +64,8 @@ module VASP {
         base_url: vector<u8>,
         compliance_public_key: vector<u8>
     ): ParentVASP {
-        // NOTE: Only callable in testnet
-        Transaction::assert(Testnet::is_testnet(), 10041);
+        // We allow invalid compliance public keys in testnet
```

Comment:
I'm confused by this comment. Isn't the check firing in testnet and mainnet?

Reply:
good spot! I wrote that then realized it should be on and forgot to remove.